### ISSUE TITLE
can i use coinbase with the bitcoin payment method?

### ISSUE THREAD

**Issue by PAOM** [agiliq/merchant#120]

**Comment by theju** [comment]
I don't think so but would wait for @tuxcanfly (our BTC guy) to confirm. I believe the bitcoin gateway in merchant talks to your local bitcoin client via JSON-RPC api.

Having said that, would you be interested in contributing a coinbase integration?

**Comment by tuxcanfly** [comment]
@theju is right, only bitcoin client is supported. I'd be in favor of replacing it with a coinbase / bitpay plugin if that would help.
